### ISSUE TITLE
Dealing with joint venture company within company field

### ISSUE THREAD

**Issue by anderspeders** [NRGI/resourcecontracts.org#790]
For contracts in Dem. Rep. Congo we have a particular issue related to the company structure behind contracts: 

The procedure is the following: 
- A series of companies *and* the national mining company forms a joitn venture company which becomes the sole owner of the contract. 

For example: 
http://www.resourcecontracts.org/contract/ocds-591adf-CD4652553506RC

**Kamoto** as joint venture is formed by Kinross (company) and Gecamines (nat mining company). 

Description: 
The Kamoto Mine is an underground copper and cobalt mine to the west of Musonoi in Katanga Province, Democratic Republic of the Congo.[1] The exploitation rights are owned by the Kamoto Copper Company (KCC), a joint venture between Katanga Mining (75%) and the state-owned Gécamines (25%).[2] 

Question: 
How can the joint venture be added in a way that accurately describes the relationship without causing major alteration of the data model / changes from the extractives extension?

cc @samCCSI @KaitlinCCSI

**Comment by SamCCSI** [comment]
Would it be possible to create an additional metadata field: "Name of the company created"? 

This would align with the annotation field with a similar name, and would allow for clarity and more accurate metadata, as opposed to just including JVs as companies (which would incorrectly imply that they are signatories), or excluding them.

**Comment by jpmckinney** [comment]
I'm not clear on the issue. @SamCCSI : You write that the JVs are not signatories. So who are the signatories?

In the Kamoto Mine example, it sounds like the JV is the signatory. In that case, it makes sense to treat it as a company. The question of company hierarchy/control is perhaps out-of-scope.

**Comment by timgdavies** [comment]
As others have noted [in the OCDS thread](https://groups.google.com/a/open-contracting.org/forum/?utm_medium=email&utm_source=footer#!msg/standard-discuss/K6DReV3SwEU/qiDzwYnbBAAJ), it is important not to confuse information about the **legal entity** that is party to the contract (the Joint Venture) with it's ownership information.

Ideally, in a world of joined up data, company hierarchy information might live outside the contracting data structures (OCDS provides a ```uri``` property in the [identifier](http://standard.open-contracting.org/latest/en/schema/reference/#identifier) building block to help making linkage between datasets), and so you might want to maintain a separate list of JVs, with structured information on their ownership.

In the event this is practically difficult, in the extensible OCDS model, you could include a nested block within ```organization``` for ```stakeholders```, and then nest organizations within that, perhaps with some meta-data to describe the nature of their stakeholding. However, modelling this to cater for all cases might be tricky - and it is something that perhaps merits wider work through Joined Up Data network or others spaces. The two starting places I would look to for such a model would be the work of Open Corporates, or the [proposed structure for ownership information in the Legal Entity Identifier (still to be clarified - see 7th Sept consultation)](http://www.leiroc.org/).

**Comment by anderspeders** [comment]
Here are my view on the options:

1) We define the JV as the legal signatory party towards the state: In this case the JV should be the sole signatory party. The rest of the company information could be added in the **notes field** as long as we do not include beneficial ownership information. 

This option seems to be rejected by Perrine.

2) We define that the companies who are party to the JV act as signatories - and that the JV is not the signatory company. This seems to be what Perrine has argued. I have noticed that the companies behind the JV are normally mentioned and appear to act as signatories. 

In this case - when the companies behind the JV act as signatories - I suggest we add the information in the **notes field** rather than creating a separate metadata field.   

3) In many cases the JV becomes the reference point for the project - thus we could apply the **Project title** field. 

4) To add a metadata field: "Name of the company created" is in my opinion a less ideal solution as it creates a specific metadata field which only applies to the situation in DRC. It is unclear to me what users will gain additionally from this rather than capturing this compared to the "notes field".

I would recommend preferably 3) or alternatively 2)

**Comment by jpmckinney** [comment]
Where is any of Perrine's content? I haven't seen it.

**Comment by KaitlinCCSI** [comment]
Perrine is not on Github, so her comments have come via email and other
conversations.

Am jumping in to respond to Anders's suggestions/comments:

Using the Kamoto example: the contract is a Convention de Joint-Venture, so
the signatory parties are Gecamines and Kinross. Kamoto (the joint venture)
is NOT a signatory, but rather the JV created through the contract.

The current metadata fields are for the "Government Entity" and the
"Company Name", and, when annotated, our annotation field for company
signatory is "Name of company executing document". My understanding is
that, to date, RC.org has treated "company name" and "name of company
executing document" as the same -- the name of the company that is a
signatory to the contract.

This is why it doesn't make sense to have Kamoto (the joint venture created
via the contract but not a signatory) listed in a field that we have been
considering as limited to the name of the contract signatory. So either we
need to come up with another place to put the JV name, or we need to
develop a new approach to how the company name/name of company executing
document fields are used, and ensure consistency throughout site.

Also, I don't see this as necessarily being specific to the DRC, because
while currently the DRC contracts might be the main ones raising this
issue, we are comfortable having joint venture agreements as a contract
type, and so presumably this issue might arise in other contexts (as
opposed to having an already formed joint venture enter into a different
type of contract, like a concession agreement -- in that case, the JV would
clearly be the signatory and thus included in the company name/name of
company executing document fields).

Does this help clarify the issues?





On Mon, Mar 7, 2016 at 12:58 PM, James McKinney <notifications@github.com>
wrote:

> Where is any of Perrine's content? I haven't seen it.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/NRGI/resourcecontracts.org/issues/790#issuecomment-193395975>
> .
> ------------------------------
>
> Spam
> <https://antispam.law.columbia.edu/canit/b.php?i=01Qr6Wnug&m=d640ec016c20&t=20160307&c=s>
> Not spam
> <https://antispam.law.columbia.edu/canit/b.php?i=01Qr6Wnug&m=d640ec016c20&t=20160307&c=n>
> Forget previous vote
> <https://antispam.law.columbia.edu/canit/b.php?i=01Qr6Wnug&m=d640ec016c20&t=20160307&c=f>
>



-- 


*KAITLIN Y. CORDES*

*Head: Land and Agriculture**Columbia Center on Sustainable Investment*
Columbia Law School - The Earth Institute, Columbia University
435 West 116th St. New York, NY 10027
| +1 (917) 658 5172
| kaitlin.cordes@law.columbia.edu
| www.ccsi.columbia.edu | @CCSI_Columbia <https://twitter.com/CCSI_Columbia>
 | @kaitlincordes <https://twitter.com/kaitlincordes>

**Comment by jpmckinney** [comment]
I think, in all cases, reality should win. If a JV is formed and then the JV signs a contract, then the JV shows up in the company field. If the JV is only formed through a contract or after a signature, then the JV does not show up in the company field, but the actual signatory.

So, there should be no hard rule about where JVs show up. They may be signatories. They may not be. You can only tell by looking at the signatories of the contract.  Going through the options:

1) I don't see why this option is rejected in general. If the JV is, in reality, the signatory, I don't see why the data shouldn't reflect reality. Maybe Perrine was only considering an option where the JV was not the signatory.

2) As with 1), if the reality is that the companies are the signatories, then use this option. If the reality is that the JV is the signatory, then don't use this option.

3) I don't think it's a good idea to overload another field or change another field's semantics. I'd prefer to throw data into a free-text notes field than to make the schema inconsistent or unsemantic.

4a) I'm not clear on the value of adding more metadata fields either.

4b) I don't think this metadata scheme should start down the road of modeling company ownership. That's adding a lot of scope, and it's not really this project's responsibility.

So, it seems what we really need is guidance on establishing what is true, what is reality, and not so much guidance on where to stick facts, because where you stick facts depends on establishing what is true.

In short, put whatever organization (or organization**s**) is actually a signatory in the company field, and put any other companies you want to track in the notes field.

**Comment by KaitlinCCSI** [comment]
Thanks, @jpmckinney, all sounds sensible. Perrine wasn't opposed to putting the JV name in as company name when the JV is the signatory; she was only considering the option where the JV is not the signatory (my understanding is that this is the situation with all of the DRC contracts?).

The only slight complication re the DRC context, which I forgot to mention before: apparently the JV name is useful for helping DRC citizens identify the underlying project, since that's the name commonly used on the ground. We are currently trying to come up with an autonaming feature that pulls from metadata. If we don't put the JV name in a metadata field that feeds into the autonaming functionality, then the DRC contract names will not include the commonly used name on the ground. This is why Perrine has requested that DRC contracts either not having the autonaming feature applied, or that we find a place to note the JV name and then a way for that info to be included in the autonaming. 

Another approach that seems sensible is to just allow for manual adjustment of the contract name in narrow contexts, like this one, but @anderspeders has been opposed to manual adjustments.

**Comment by jpmckinney** [comment]
As a compromise for autonaming, could there be a new field added, along the lines of "autonaming affix", which would allow editors to add some additional text to be prefixed, suffixed or otherwise affixed to the autoname? That way, autonaming is still automatic, but allows the addition of other text that isn't sourced from the standard fields.

**Comment by KaitlinCCSI** [comment]
That sounds like an excellent option. @anderspeders, might this proposal for autonaming work for you?

**Comment by anderspeders** [comment]
Happy to go with manual edits of auto-naming (I believe we've already discussed this on a call).

@KaitlinCCSI: In terms of this specific issue re. JVs - should we go with @jpmckinney's summary above and add this to the Administrator Guide following either option 1) or 2) outline above based on the reality of the situation.

**Comment by SamCCSI** [comment]
Confirming @anderspeders 's proposal (_put whatever organization (or organizations) is actually a signatory in the company field, and put any other companies you want to track in the notes field._)

To be more explicit, the rule will be: Where there is a joint venture (JV) involved in the agreement, the JV should only be listed as a signatory where the JV's own representative signs the contract. Where the contract is instead signed by the corporate parties participating in the joint venture, rather than the JV itself, then only those corporate parties should be listed as company signatories.

**Comment by anderspeders** [comment]
Excellent, 

@SamCCSI: Could you possibly add this to the documentation? 

We can then close this ticket.

**Comment by SamCCSI** [comment]
@anderspeders done.

**Comment by anderspeders** [comment]
Fantastic. Closing issue.
